source/code-snippets/typescript/dot-notation.ts

```diff
@@ -15,11 +15,11 @@ const collection = database.collection<TestType>("<your collection>");
 await collection.updateOne({}, { $set: { field: { nested: "A string" } } });
 // end-error
 // start-no-key
-interface TestNumber {
-  myNumber: number;
+interface Furniture {
```

Comment:
Suggestion: choose an object that is a countable noun rather than an uncountable one. This can help a reader build an understanding of its contents more quickly as well as make it easier to translate. A simple countable object to understand might be:
```
interface User {
  email: string;
}
```

Reply:
Implemented the suggested change.

I'm not sure I see the benefit here of changing `Furniture` to `User` here ("pieces of furniture" and "users" are both able to be counted, and I would argue "Furniture" is more well known). If there is anything else suggesting `Furniture` was unacceptable here please let me know.